versioned/persist/serialize/src/main/proto/persist.proto

```diff
@@ -77,10 +77,34 @@ message GlobalStateLogEntry {
   repeated ContentIdWithBytes puts = 4;
 }
 
+message RefLogEntry {
+  bytes ref_log_id = 1;
+  bytes ref_name = 2;
+  enum RefType {
+    Branch = 0;
+    Tag = 1;
+  }
+  RefType ref_type = 3;
+  bytes commit_hash = 4;
+  repeated bytes parents = 5;
+  int64 operation_time = 6;
+  enum Operation {
+    CREATE_REFERENCE = 0;
+    COMMIT = 1;
+    DELETE_REFERENCE = 2;
+    ASSIGN_REFERENCE = 3;
+    MERGE = 4;
+    TRANSPLANT = 5;
+  }
+  Operation operation = 7;
+  bytes details = 8;
```

Comment:
None of the implementations add any value to this property - i.e. all information in "details"  is available via the other fiels. Really no need to keep this property. It would be okay to add the merged/transplanted hashes as a separate, optional attribute. But I'm -1 on persisting a human readable message that doesn't really add value.